### ISSUE TITLE
LateInitializationError when decoding with XZDecoder

### ISSUE THREAD

**Issue by Quijx** [brendan-duncan/archive#171]
I tried to use `XZDecoder.decodeBytes` and got the following error:
```
LateInitializationError: Field '_input@70520520' has already been initialized.
#0      LateError._throwFieldAlreadyInitialized (dart:_internal-patch/internal_patch.dart:211:5)
#1      RangeDecoder._input= (package:archive/src/lzma/range_decoder.dart:33:30)
#2      RangeDecoder.input= (package:archive/src/lzma/range_decoder.dart:45:39)
#3      LzmaDecoder.decode (package:archive/src/lzma/lzma_decoder.dart:142:12)
#4      _XZStreamDecoder._readLZMA2 (package:archive/src/xz_decoder.dart:287:26)
#5      _XZStreamDecoder._readBlock (package:archive/src/xz_decoder.dart:143:5)
#6      _XZStreamDecoder.decode (package:archive/src/xz_decoder.dart:59:7)
#7      XZDecoder.decodeBuffer (package:archive/src/xz_decoder.dart:22:20)
#8      XZDecoder.decodeBytes (package:archive/src/xz_decoder.dart:17:12)
```

**Comment by robert-ancell** [comment]
Duplicate of https://github.com/brendan-duncan/archive/issues/209

**Comment by brendan-duncan** [comment]
Fix is in 3.3.6
